packages/aws-client/src/test-utils.ts

```diff
@@ -23,7 +23,7 @@ const localStackPorts = {
   firehose: 4566,
   iam: 4566,
   kinesis: 4566,
-  kms: 4566,
+  KMS: 4566,
```

Comment:
What was the reason for this change? Well

Reply:
this is going to be looked up by string (we don't do any caps manipulation) based on the service.serviceIdentifier or service.name parameters, which is declared in @aws-sdk/client-kms
https://github.com/nasa/cumulus/pull/3541/files does the same thing for the same reason